bin/cli.js

```diff
@@ -1,15 +1,14 @@
 #!/usr/bin/env node
 
-'use strict';
+/* eslint no-console: "off" */
 
-var fs = require('fs');
-var chalk = require('chalk');
-var program = require('commander');
-
-var StyleStats = require('../lib/stylestats');
-var Format = require('../lib/format');
-var specs = require('../lib/specs');
-var util = require('../lib/util');
+const fs = require('fs');
+const chalk = require('chalk');
+const program = require('commander');
+const StyleStats = require('../lib/stylestats');
+const Format = require('../lib/format');
+const specs = require('../lib/specs');
+const util = require('../lib/util');
```

Comment:
`util` and `fotmat` should be exposed as property of `stylestats`.
This requires refactoring.